_posts/2021-01-28-release-0-21-6.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "JuMP 0.21.6 is released"
+date:   2021-01-28
+categories: [releases]
+permalink: /blog/0.21.6-release/
+---
+
+We are happy to announce the release of JuMP 0.21.6. This should be a
+non-breaking release, please let us know if this isn't the case by
+[opening a Github issue](https://github.com/jump-dev/JuMP.jl/issues/new/choose)
+or joining the [developer chatroom](https://gitter.im/JuliaOpt/JuMP-dev).
+
+A key feature of interest is that we have significantly revised our
+documentation, including [examples that are now build in the documentation](https://jump.dev/JuMP.jl/v0.21.6/examples/basic/).
+We hope to make much more progress improving the documentation over the coming
+months, so if you have suggestions for how things could be improved, please
+reach out.
+
+A summary of the other changes are as follows:
+
+- New features:
+  * Added support for skew symmetric variables via
+    `@variable(model, X[1:2, 1:2] in SkewSymmetricMatrixSpace())`.
+  * `lp_sensitivity_report` has been added which significantly improves the
+    performance of querying the sensitivity summary of an LP.
+    `lp_objective_perturbation_range` and `lp_rhs_perturbation_range` are
```

Comment:
It's a bit late now, but now I remember we made a big fuss when one of our dependencies deprecated a function in a patch release. We should be mindful of this until we drop support for the Julia releases that display deprecation warnings by default.

Reply:
Agreed on not adding deprecations in general in a patch release. The justification for doing it in this case was:
- The current implementation is niche, and too slow for anything other than toy models, so it probably wasn't used by another package (no one had complained).
- Both functions still work, they just have the following at the start
   ```Julia
    @warn(
        "This function has been deprecated and will be removed in a future " *
        "release. Use `lp_sensitivity_report(model)` instead.",
        maxlog = 1,
    )
    ```
We can remove the warning and release 0.21.7 if people complain?